doc/api/next_api_changes/deprecations/20105-AL.rst

```diff
@@ -0,0 +1,3 @@
+ArrowStyle.Curve attributes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The ``
```

Comment:
Incomplete...

Reply:
oops, removed (that is a remnant of when I was thinking of removing the attributes on Curve.